server/controllers/finance/reports/recovery_capacity/index.js

```diff
@@ -39,6 +43,16 @@ async function report(req, res, next) {
     const formatedDateFrom = moment(dateFrom).format('YYYY-MM-DD');
     const formatedDateTo = moment(dateTo).format('YYYY-MM-DD');
 
+    const daysWeek = [
```

Comment:
It seems like this should be in some common file for access by any report on the server?